webapp/components/user_settings/custom_theme_chooser.jsx

```diff
@@ -108,15 +108,24 @@ class CustomThemeChooser extends React.Component {
         this.onPickerChange = this.onPickerChange.bind(this);
         this.pasteBoxChange = this.pasteBoxChange.bind(this);
         this.toggleContent = this.toggleContent.bind(this);
+        this.onCodeThemeChange = this.onCodeThemeChange.bind(this);
 
         this.state = {};
     }
+
     componentDidMount() {
         $('.color-picker').colorpicker({
             format: 'hex'
         });
         $('.color-picker').on('changeColor', this.onPickerChange);
+        $('.group--code').on('change', this.onCodeThemeChange);
+    }
+
+    componentWillUnmount() {
+        $('.color-picker').off('changeColor', this.onPickerChange);
+        $('.group--code').off('changeColor', this.onCodeThemeChange);
```

Comment:
I think this should be
`$('.group--code').off('change', this.onCodeThemeChange);`
right?